### DISCUSSION TITLE
Remove the ability to paste in the access and override prompts

### DISCUSSION THREAD

**Discussion by lenehc** [proginosko/LeechBlockNG#467] [Ideas]
By using the developer console, it is possible to copy and paste the randomly generated access and override code into the prompt. This could be fixed by simply preventing the default action of "onpaste", detailed [here](https://medium.com/@clairecodes/how-to-prevent-pasting-into-input-fields-79476b388b75)

![Screenshot](https://github.com/proginosko/LeechBlockNG/assets/115498815/f0603619-b121-4c2e-9d6a-f98e07055a9c)

**Comment by proginosko** [comment]
Thanks for this. The simpler solution is to select the option "Display access code as image" (Options > General > Miscellaneous).

**Reply by lenehc** [reply]
Thanks!
